go/service/chat_local.go

```diff
@@ -537,7 +539,7 @@ func (h *chatLocalHandler) getConversationInfo(ctx context.Context,
 	conversationInfo.TopicType = conversationRemote.Metadata.IdTriple.TopicType
 
 	if len(conversationRemote.MaxMsgs) == 0 {
-		return chat1.ConversationInfoLocal{}, nil,
+		return conversationInfo, maxMessages,
```

Comment:
Is there a reason to return existing variables here? I just changed all of them to newly constructed zero values per patrick's suggesting in [a previous PR](https://github.com/keybase/client/pull/4405#pullrequestreview-1791423)